CODEOWNERS

```diff
@@ -0,0 +1,18 @@
+# Default owners for the entire repository
+default_owners:
+  - svyas@redhat.com
+  - arxcruz@redhat.com
+
+# Owners for specific directories within the repository
+directory_owners:
+  # Component directory "Heat"
+  heat:
+    owners:
+      - rabi@redhat.com
+      - bshephar@redhat.com
+
+  # Component directory "Neutron"
+  neutron:
+    owners:
+      - karelyatin@redhat.com
+      - slawqo@redhat.com
```

Comment:
```suggestion
# Default owners for the entire repository
* @son-vyas @arxcruz 

# Component directory "Heat"
/heat/ @rabi @bshephar 

# Component directory "Neutron"
/neutron/ @karelyatin @slawqo 
```

Reply:
but the owners file would be different here and release - https://github.com/openshift/release/pull/53577/files. Because in this repository there are no prow jobs running but in release we have and prow jobs dont allow the above way - https://www.kubernetes.dev/docs/guide/owners/#cnx-opp-owners